blog/_posts/2019-01-18-fluxdiffeq.md

```diff
@@ -247,18 +209,26 @@ as `θ` in associated literature). The utility of this will be seen later.
 
 ## Let's Put an ODE Into a Neural Net Framework!
 
-To understand embedding an ODE into a neural network, let's recall what a layer
-of a neural network actually is. A layer is a transformation 𝐑ⁿ→𝐑ᵐ which
-takes in a vector of size `n` and spits out a new vector of size `m`. That's it!
-So a differential equation layer is simply a layer that takes in a parameter
-vector `p` (of size `n`) and spits out a new vector (of size `m`). You can do this
-manually using the
-[`remake` function in DifferentialEquations.jl](http://docs.juliadiffeq.org/latest/basics/problem.html#Modification-of-problem-types-1),
-or you can use the Flux.jl layers defined in DiffEqFlux.jl. Let's give those a spin.
-
-The most basic differential equation layer is `diffeq_rd`. The function is of
-the form `diffeq_rd(p,reduction,prob,args...;kwargs...)`. The way to understand this
-function is that it does the following. It takes in the parameter vector `p`,
+To understand embedding an ODE into a neural network, let's look at what a
+neural network layer actually is. A layer is really just a *differentiable

```

Comment:
well, whether differentiability is really necessary is debatable. We can keep this in though